Docs/development/ui.md

```diff
@@ -129,12 +129,12 @@ So far you have SingularityUI being served by Brunch, and SingularityService run
 
 Open up SingularityUI in your browser by going to [`http://localhost:3333`](http://localhost:3333).
 
-You'll be prompted to input an API root. This is the service that SingularityUI will interact with for its data. Give it your `http://example/singularity/api` URI.
+You'll be prompted to input an API root. This is the service that SingularityUI will interact with for its data. Give it your `http://192.168.59.103/singularity/api` URI.
```

Comment:
might simplify things to instruct users to add the output of `boot2docker ip` to `/etc/hosts` as `docker`